Project.toml

```
@@ -12,6 +12,19 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 
 [compat]
 Distributions = "0.25"
-MLJBase = "0.18, 0.19, 0.20, 0.21"
-MLJTuning = "0.6, 0.7"
-julia = "1.6"
+LinearAlgebra = "1.11"
```

LinearAlgebra is a standard library. You should make this change and that should fix the LTS fail:

```suggestion
LinearAlgebra = "1"
```